### DISCUSSION TITLE
Clevo / Gigabyte G5 Fan Support

### DISCUSSION THREAD

**Discussion by Pyro1992** [Rem0o/FanControl.Releases#2160] [Ideas]
Hey,

it would be nice if you can add support for the Gigabyte G5 2023 that uses "Clevo" Software for Fan control. Also with your Fan Control it doesn´t work / detects any Sensor.

This tool work for changing fan speed > https://github.com/zuyan9/RLECViewer (Clevo custom fan control)  <<< little bit buggy, below a better ones

Better ones > https://github.com/djsubtronic/ClevoFanControl

**Comment by Rem0o** [comment]
FanControl has a plugin (https://github.com/Rem0o/FanControl.Releases/wiki/Plugins)  system to add sensors into the software.   If you have the time to wrap/port the code over a plugin, it could add support.
